@xen-orchestra/babel-config/index.js

```diff
@@ -23,6 +23,9 @@ const configs = {
 
       shippedProposals: true,
       targets: (() => {
+        if (!__PROD__) {
+          return { firefox: '80', chrome: '84', node: 'current' }
+        }
```

Comment:
I'm uncomfortable with this change because there might have different behaviors between dev and prod compilation mode.

I believe we did it before but we removed it.

What package precisely is taking too much time to compile for you?

Reply:
Everything is taking 2x more time without this. Could we at least have a flag for this?

Reply:
You should avoid doing a `yarn run dev` for the whole project, we might even remove this in the future.

Simply do it for the modules you are working on :slightly_smiling_face:

Reply:
Also most packages set node versions to >= 8 which makes all features to be compiled for 8, instead of required node 12

Reply:
Indeed, I'm increasing them progressively for the products (applications & plugins).

Regarding libraries, I don't increase the requirement unless necessary.

I've chosen this version because it supports most of the stuff that require heavy transpiling, feel free to ping me if you think a change is necessary for a package.

Reply:
the slowdown is not only with dev, but also tests (babel-jest is used), and production build. I happen to work in docker on macos so the pain is doubled. I understand you won't merge this but some option to compile only for latest browsers and used node version would be good. Even self-hosted production build could use this (I know I'll use node 12, and I know I'll use just latest firefox and chrome in production). Maybe e.g. `TARGETS` env variable could be respected?

Reply:
> Maybe e.g. `TARGETS` env variable could be respected?

I'm not opposed to this :slightly_smiling_face: